index.html

```diff
@@ -0,0 +1,671 @@
+    <nav class="top-bar navbar navbar-expand-lg navbar-dark justify-content-between align-items-center flex-nowrap">
+      <div class="navbar-brand mr-auto">
+        <a href="http://test.com">
+          <img class="d-lg-none" alt="Article Image" src="img/logo-small.png">
+        </a>
+        <div class="weather-date d-none d-lg-block">
+          <p class="weather"> <i class="fas fa-water"></i> 30° India</p>
+          <p class="date">Fri, Sep 18, 2020</p>
+        </div>
+      </div>
+      <div>
+        <a href="http://test.com">
+        <img class="d-none d-lg-block" alt="Article Image" src="img/logo-large.png">
+        </a>
+        </div>
+      <div class="side-nav d-flex flex-row justify-content-around align-items-center ml-auto">
+        <i class="fas fa-user"></i>
+        <button class="btn btn-warning">Subscribe &gt; </button>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#toggle-menu" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+      </div>
+    </nav>
+    <div class="bottom-menu d-none d-lg-flex navbar collapse navbar-collapse w-100 flex-row justify-content-start" id="toggle-menu">
+      <ul class="bottom-list d-none d-lg-flex navbar navbar-nav flex-row justify-content-around">
+        <li class="nav-item"><a class="nav-link" href="#">U.S.</a></li>
+        <li class="nav-item"><a class="nav-link" href="#">World</a></li>
+        <li class="nav-item"><a class="nav-link" href="#">Business</a></li>
+        <li class="nav-item"><a class="nav-link" href="#">Tech & Science</a></li>
+        <li class="nav-item"><a class="nav-link" href="#">Culture</a></li>
+        <li class="nav-item"><a class="nav-link" href="#">Newsgeek</a></li>
+        <li class="nav-item"><a class="nav-link" href="#">Sports</a></li>
+        <li class="nav-item"><a class="nav-link" href="#">Health</a></li>
+        <li class="nav-item"><a class="nav-link" href="#">The Debate</a></li>
+        <li class="nav-item"><a class="nav-link" href="#">Vantage</a></li>
+        <li class="nav-item"><a class="nav-link" href="#">Weather</a></li>
+      </ul>
+      <div class="search">
+        <form>
+          <div class="input-group">
+            <input type="text" class="form-control" placeholder="Search" name="search">
+            <div class="input-group-btn">
+              <button class="btn btn-default" type="submit"><i class="fa fa-search"></i></button>
+            </div>
+          </div>
+        </form>
+      </div>
+    </div>
+  </header>
+  <main>
+    <section class="big-grid">
+      <div class="col1 container">
+        <div class="col-title">FEATURED STORIES</div>
+        <article class="story container-sm d-flex flex-column justify-content-start">
+          <div class="img-w-sub">
+            <div class="inner-subtitle d-block">
+              <a href="#">MY TURN</a>
+            </div>
```

Comment:
[OPTIONAL] Since the inner title has the same background as the website background, the image seems cut-out. You could add a little translucent effect to the inner title background similar to the official page.